app/celery/broadcast_message_tasks.py

```diff
@@ -70,6 +75,7 @@ def send_broadcast_provider_message(broadcast_event_id, provider):
             previous_provider_messages=broadcast_event.get_earlier_provider_messages(provider),
             sent=broadcast_event.sent_at_as_cap_datetime_string,
             expires=broadcast_event.transmitted_finishes_at_as_cap_datetime_string,
+            channel=channel
```

Comment:
i wonder what happens if this channel is different to the initial broadcast's channel? i dont think we necessarily need to change anything, but i wonder about the relative merits of re-reading it here rather than pulling the data out of the original `alert` (somehow - we don't currently store the channel it went out on 🤔)

Reply:
Good spot. I might leave an inline comment here about this.

Note, we did have a brief discussion the other day about storing the channel on the broadcast so we would be able to reference it always from there but decided given the channel is unlikely to change, we should just go for the quicker option, at least for now and just take it from the service every time